docs/high-availability/provision-flannel.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function setup_flannel {
+    yum install -y flannel
```

Comment:
ok, ill make it consistent.  Given that we're just "Getting lucky" with the fact that vagrant does this for us, I think its good to have the sudo in there for the commands where its obviously required.